### PR TITLE
Extract Maya Scene Raw loaded container would check on NoneType value.

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
This PR is to fix the publishing error occurs when we load the meshes and publish them as part of the layout product type.
The error shown below:
image

## Additional info
n/a

## Testing notes:

1. Launch Maya
2. Load assets with model product type
3. Create Layout with the loaded asset
4. Publish

### CODE REVIEW

File: client/ayon_maya/plugins/publish/extract_maya_scene_raw.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def _get_loaded_containers(members):
                 continue
 
             set_content = set(cmds.sets(obj_set, query=True))
```
Comment (BigRoy) [inline client/ayon_maya/plugins/publish/extract_maya_scene_raw.py:147]:
```suggestion
            set_content = set(cmds.sets(obj_set, query=True) or [])
```

File: client/ayon_maya/plugins/publish/extract_maya_scene_raw.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def _get_loaded_containers(members):
...
-            if set_content.intersection(members_with_refs):
+            if set_content and set_content.intersection(members_with_refs):
```
Comment (BigRoy) [inline client/ayon_maya/plugins/publish/extract_maya_scene_raw.py:148]:
```suggestion
            if set_content.intersection(members_with_refs):
```

**Review by BigRoy — CHANGES_REQUESTED** [review]
You code doesn't fix the error because the error is on the unchanged line above. I've proposed a fix.

**Review by BigRoy — APPROVED** [review]
Didn't test but change makes sense. :)

**Comment by BigRoy** [comment]
Not needed for this "hotfix" but this should preferably use the `lib.get_container_members` function call?
AND also, this may be much better if it would also use the actual `host.get_containers()` method to ensure it actually collects containers that the Maya integration itself also defines are containers instead of the custom logic here.

**Comment by moonyuet** [comment]
> Not needed for this "hotfix" but this should preferably use the `lib.get_container_members` function call? AND also, this may be much better if it would also use the actual `host.get_containers()` method to ensure it actually collects containers that the Maya integration itself also defines are containers instead of the custom logic here.

maybe we should use the one from library instead. Much safer and make sense

File: client/ayon_maya/plugins/publish/extract_maya_scene_raw.py [hunk 3]
```diff
@@ -79,7 +78,7 @@ def process(self, instance):
         if set(self.add_for_families).intersection(
                 set(instance.data.get("families", []))) or \
                 instance.data.get("productType") in self.add_for_families:
-            selection += self._get_loaded_containers(members)
+            selection += get_container_members(members)
```
Comment (BigRoy) [inline client/ayon_maya/plugins/publish/extract_maya_scene_raw.py:81]: This change is wrong.. we still need to first get the containers that are members of this instance... and then go from there. The logic you have here now will error.

**Review by BigRoy — APPROVED** [review]
Looks good to me.

**Pull request merged**